content/300-guides/200-deployment/201-serverless/300-deploy-to-vercel.mdx

```diff
@@ -6,281 +6,47 @@ metaDescription: 'Learn how to deploy a Next.js application based on Prisma Clie
 
 <TopBlock>
 
-This guide takes you through the steps to set up and deploy a serverless application that uses Prisma to [Vercel](https://vercel.com/). The application uses Next.js to provide a serverless REST API, and Prisma Client to handle fetching, creating, and deleting records from a database.
+This guide takes you through the steps to set up and deploy a serverless application that uses Prisma to [Vercel](https://vercel.com/).
 
 Vercel is a cloud platform that hosts static sites and serverless functions. You can integrate a Vercel project with a GitHub repository to allow you to deploy automatically when you make new commits. [Next.js](https://nextjs.org/) is a full-stack React-based serverless framework that integrates seamlessly with Vercel.
 
-The Next.js app has the following components:
-
-- **Backend**: Next.js API routes that serve a REST API and use Prisma Client to handle database operations on a PostgreSQL database.
-- **Frontend**: Next.js page to interact with the API.
+We created an [example application](https://github.com/prisma/deployment-example-vercel) using Next.js you can use as a reference when deploying an application using Prisma to Vercel.
 
 ![Prisma Vercel deploys - architecture diagram](./images/300-00-deploy-to-vercel-architecture.png)
 
-This guide focuses on how to use Prisma with Next.js and Vercel. The starting point is the [Prisma Vercel example](https://github.com/prisma/deployment-example-vercel), which has four serverless REST endpoints configured as [Next.js API Routes](https://nextjs.org/docs/api-routes/introduction) and a Next.js page that allows you to call the endpoints and display data.
-
 </TopBlock>
 
 ## Prerequisites
```

Comment:
Feels like this is not really useful any more. There are no steps to follow, so nothing to do "pre" following them.

The first sentence could be the text under a "Deploy an application to Vercel" headline, where we then explain that an app using Prisma is not that different from any other app - and then we could link to some Vercel guides for common apps.

Reply:
Removed